### DISCUSSION TITLE
vim-airline

### DISCUSSION THREAD

**Discussion by LuisMalave2001** [JetBrains/ideavim#287] [Ideas]
It just a suggestion... It would be worth if vim-airline features are supported or something similar.

**Comment by AlexPl292** [comment]
Hi! Could you please describe what features do you miss from vim-airline?

**Reply by LuisMalave2001** [reply]
Customization, for example,
I used to toggle a lot the underscore character "_" as keyword using
So, I added a this to my vimrc to put a little helper message
![image](https://user-images.githubusercontent.com/58850261/114554545-20c0c180-9c67-11eb-913a-436d7af8e1c8.png)

You can add any section you want as you need it.

**Comment by ManuelTS** [comment]
I gladly bring this discussion back from the dead for buffer numbers in the left of the buffer name (or tab / window in Intellij) corresponding with the enumeration one sees when entering `:ls`. I would love to simply use `:b<number>` without memorizing some numbers to quickly switch fast between files in Intellij as in vim with airline.

Airline offers this options by enabling
```
let g:airline#extensions#tabline#show_buffers = 1
let g:airline#extensions#tabline#buffer_nr_show = 1
```
Only relying on `:b#`, `:bp`, `:bn`, and so forth is insufficient for me and other people too programming on complex projects.
